File: app/src/main/java/br/com/kakobotasso/lodjinha/models/Categoria.java
package br.com.kakobotasso.lodjinha.models;

import com.google.gson.annotations.SerializedName;

public class Categoria {
    @SerializedName("id")
    int id;

    @SerializedName("descricao")
    String descricao;

    @SerializedName("urlImagem")
    String imagem;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public String getImagem() {
        return imagem;
    }

    public void setImagem(String imagem) {
        this.imagem = imagem;
    }
}

File: app/src/main/java/br/com/b2w/desafio/model/categoria/Categoria.kt
package br.com.b2w.desafio.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Categoria (
    @SerializedName("id") val id: Int,
    @SerializedName("descricao") val descricao: String,
    @SerializedName("urlImagem")val urlImagem: String
) : Parcelable